src/bot.ml

```diff
@@ -192,6 +192,63 @@ let callback _conn req body =
             |> Lwt.async ;
             Server.respond_string ~status:`OK ~body:"Handling minimization." ()
             )
+          else if
+            string_match
+              ~regexp:
+                (f "@%s:? [Cc][Ii][- ][Mm]inimize:?\\( [^\n]*\\)?" bot_name)
+              body
+          then (
+            let requests =
+              String.split ~on:' '
+                (Stdlib.String.trim
+                   (* FIXME why not `String.trim`? ocamlc complains it's unbound... (cf https://coq.zulipchat.com/#narrow/stream/243318-coqbot-devs.20.26.20users/topic/.60String.2Etrim.60/near/236752334) *)
+                   (Str.global_replace (Str.regexp "[ ,]+") " "
+                      (Str.matched_group 1 body)))
```

Comment:
Would be more readable if written with pipes (`|>`).

Reply:
Done